Macos build fix (#205)
Concordium/concordium-node: concordium-node/README.md

## Dependencies to build the project
* Rust (stable 1.53 for using static libraries)
* binutils >= 2.22

>   * for macOS, install Xcode and use what is included with that instead, as these are tailored to the OS version.

Reviewer: ```suggestion
  * For macOS one should use the binutils provided by Xcode.```
